site/content/docs/4.3/components/navs.md

```diff
@@ -586,24 +592,39 @@ Activates a tab element and content container. Tab should have either a `data-ta
 </div>
 
 <script>
-  $(function () {
-    $('#myTab li:last-child a').tab('show')
+  window.addEventListener('load', function () {
```

Comment:
Is this really needed?

Reply:
No it was just to match the previous example

Reply:
Maybe we should remove it then?

Reply:
sure 👍